Fix version so apk-tools understand the .so version
(1.0.0.rc4 -> 1.0.0_rc4)

diff --git a/configure.ac b/configure.ac
index 498d69a..f52bf8c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,7 +8,8 @@ m4_define([lxc_version_beta], [rc4])
 
 m4_define([lxc_version_base], [lxc_version_major.lxc_version_minor.lxc_version_micro])
 m4_define([lxc_version],
-	  [ifelse(lxc_version_beta, [], [lxc_version_base], [lxc_version_base.lxc_version_beta])])
+	  [ifelse(lxc_version_beta, [], [lxc_version_base],
+	  [lxc_version_base[_]lxc_version_beta])])])
 
 AC_INIT([lxc], [lxc_version])
 
